regolith/helpers/l_grantshelper.py

```diff
@@ -73,37 +62,26 @@ def construct_global_ctx(self):
         gtx["str"] = str
         gtx["zip"] = zip
 
-
     def sout(self):
         rc = self.rc
-        bad_stati = ["finished", "cancelled", "paused", "back_burner"]
-        projecta = []
-        if rc.lead and rc.person:
-            raise RuntimeError(f"please specify either lead or person, not both")
-        for projectum in self.gtx["projecta"]:
-            if rc.lead and projectum.get('lead') != rc.lead:
-                continue
-            if rc.person:
-                if isinstance(rc.person, str):
-                    rc.person = [rc.person]
-                good_p = []
-                for i in rc.person:
-                    if not projectum.get('group_members'):
-                        continue
-                    if projectum.get('group_members') and i not in projectum.get('group_members'):
-                        continue
-                    else:
-                        good_p.append(i)
-                if len(good_p) == 0:
-                    continue
-            if not rc.stati and projectum.get('status') in bad_stati:
+        # print('rc: {}'.format(rc))
+        # print('rc.lead: {}'.format(rc.list))
+        # print('rc.admin: {}'.format(rc.admin))
+        grants = []
+        if rc.list and rc.admin:
```

Comment:
if you require these then specify them as positional args not optional args.  Then you can remove this